lib/nostrum/struct/component/default_value.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Nostrum.Struct.Component.DefaultValue do
+  defstruct [
+    :id,
+    :type,
+  ]
```

Comment:
```suggestion
  @derive Jason.Encoder
  defstruct [
    :id,
    :type,
  ]
```
Doesn't this need to be added here?